ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/PoAAccess.java

```diff
@@ -0,0 +1,37 @@
+package de.adorsys.ledgers.baam.db.domain;
+
+import jakarta.persistence.Entity;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Entity
+public class PoAAccess extends BankAccountAccess {
+
+    // Getters and Setters
+    @Getter
+    @Setter
+    private String attorneyInFact;
+    @Getter
+    @Setter
+    private LocalDateTime validFrom;
+    @Getter
+    @Setter
+    private LocalDateTime validUntil;
+    @Getter
+    @Setter
+    private PoAType poaType;
+    private double weight;
+
+    public PoAAccess() {
+
+    }
+
+    public boolean isActive() {
+        LocalDateTime now = LocalDateTime.now();
+        return now.isAfter(validFrom) && now.isBefore(validUntil);
+    }
+}
+
+
```

Comment:
Please use the `Getters and Setter annotation properly`

you have `lombok` already, make good use of it

you are doing a great job but your code can be done in a better way.

you can refer to these : https://medium.com/@wandercosta/lombok-getters-and-setters-10ce392188f6